docs/architecture/cluster-services.rst

```diff
@@ -23,41 +22,40 @@ metrics collection, tracing, node health checking and more.
 .. _Kops: https://github.com/kubernetes/kops/
 .. _Amazon AWS: https://aws.amazon.com
 
-In MetalK8s_, we augment a basic Kubernetes cluster deployed using the
-Kubespray_ playbook) with various tools to bring an on-premise cluster to the
-same level of operability.
+MetalK8s_ augments a basic Kubernetes cluster deployed using the
+Kubespray_ playbook with tools that bring an on-premise cluster to
+the same level of operability.
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _Kubespray: https://github.com/kubernetes-incubator/kubespray/
 
-Basic Cluster Addons
+Basic Cluster Add-ons
 --------------------
-On top of the basic Kubernetes services, the following addons are deployed:
+The following add-ons are deployed atop the basic Kubernetes services:
```

Comment:
`atop`?

Reply:
"On top of" is an inherently ambiguous phrase. It can mean A) "In addition to" or B) "physically or metaphorically resting on." 
So in the present instance, it could mean:
A) "In addition to basic K8s services, [whatever it is] also deploys the following add-ons..." or 
B) "When you deploy the following add-ons, they run on the basic K8s services..."

I picked B. Did I pick wrong?